TasteBuddiesServer/src/main/java/org/launchcode/TasteBuddiesServer/models/Event.java

```diff
@@ -1,28 +1,87 @@
 package org.launchcode.TasteBuddiesServer.models;
 
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
 import javax.persistence.Entity;
 import javax.persistence.JoinColumn;
+
 import javax.persistence.OneToMany;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
+
 @Entity
 public class Event extends AbstractEntity {
+
+    @NotNull
+    private String entryCode;
+
+    @NotNull
     private String location;
+
+    @NotNull
     private String searchRadius;
-    private String entryCode;
-    @OneToMany
-    @JoinColumn(name = "event_id")
-    private List<UserLikes> userLikedRestaurants = new ArrayList<>();
-    @OneToMany
-    @JoinColumn(name = "event_id")
+
+    //fetchtype to "LAZY" for performance
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id")
+    private User user;
```

Comment:
This is a design question I had. Do we need to track which user created the event? Maybe they get edit permissions?